src/cli/commands/preview.ts

```diff
@@ -7,7 +7,7 @@ export class Preview extends Command {
     trunk: flags.string({
       default: '',
       description:
-        'State which branch is trunk. Defaults to honuring the "base" branch setting in the GitHub repo settings.',
+        'State which branch is trunk. Defaults to honouring the "base" branch setting in the GitHub repo settings.',
```

Comment:
I tend to favor `honor` over `honour`, but I think this repo favours the extra `u`.

Reply:
It might be more OSS friendly to go US Spelling here though.

Reply:
If you're up for it, I'm happy to make the change. I generally make the smallest change possible (my PRs are already quite large, so I try not to make waves).

Reply:
Yeah let's do it :)